README.PT.md

```
@@ -10,14 +10,14 @@
 <p align="center" style="font-size: 12px;">
-    <a href="https://docs.langinfra.org" style="text-decoration: underline;">Docs</a> -
+    <a href="https://docs-langinfra.khulnasoft.com" style="text-decoration: underline;">Docs</a> -
     <a href="https://discord.com/invite/EqksyE2EX9" style="text-decoration: underline;">Junte-se ao nosso Discord</a> -
     <a href="https://twitter.com/langinfra_ai" style="text-decoration: underline;">Siga-nos no X</a> -
     <a href="https://huggingface.co/spaces/Langinfra/Langinfra" style="text-decoration: underline;">Demonstração</a>
 </p>
 
 <p align="center">
-    <a href="https://github.com/khulnasoft/langinfra">
+    <a href="https://github.com/langinfra/langinfra">
         <img src="https://img.shields.io/github/stars/khulnasoft/langinfra">
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)